rapids/rapids.sh

```diff
@@ -59,10 +54,45 @@ function execute_with_retries() {
   return 1
 }
 
+function install_dask_rapids() {
+  local base
+  base=$(conda info --base)
+  local pinned
+  pinned=${base}/conda-meta/pinned
+  local mamba_env
+  mamba_env=mamba
```

Comment:
We need to do declaration/assignment split only when variable initialized with inline command execution ("`$(...)`"), so we can revert back these variables to inline definition with `local -r ...`.

Reply:
Ack, thanks for clarifying